src/components/RegistrationForm/Form/FieldValidation/FieldValidation.jsx

```diff
@@ -0,0 +1,23 @@
+import { TextValidation } from './FieldValidation.styled.js';
+import PropTypes from 'prop-types';
+
+export const FieldValidation = ({ $fieldError, $fieldName }) => {
+  return (
+    <div
+      style={{
+        height: '20px',
+        display: 'flex',
+        alignItems: 'center',
```

Comment:
It could be better without inline styles. Inline styles without exactly reasons are almost always - bad practice

Reply:
This is a draft version of the code.